upstash/data_source_upstash_cluster.go

```diff
@@ -0,0 +1,93 @@
+package upstash
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func dataSourceCluster() *schema.Resource {
```

Comment:
what will be definition in tf file for the kafka cluster?
```
resource "upstash_cluster" "mycluster" {
...
...
}
```
we might use `upstash_kafkacluster` or `upstash_kafka_cluster`